test/rules/avoid_implementing_value_types.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N avoid_implementing_value_types`
+
+class Size {
+  final int inBytes;
+  const Size(this.inBytes);
+
+  @override
+  bool operator==(Object o) => o is Size && o.inBytes == inBytes;
```

Comment:
"inBytes" --> "other.inBytes" (though it doesn't actually affect the tests :-).

Reply:
Done!